### PR TITLE
adds two new audible emotes and adds sound to others

### PR DESCRIPTION
**Pull request by maxion12345**
# Document the changes in your pull request

added two new emotes because I felt like coding all of a sudden
also I just wanted *lipsmack what a rotten way to die
also added sounds to other emotes

https://user-images.githubusercontent.com/23666142/136283001-4b6c5d51-4524-4fb8-b25c-aa40ff9dd835.mp4

(enable volume)

# Wiki Documentation
n/a

# Changelog

Edit the changelog below to reflect the changes made by this PR, even if the changes are minor - required for every PR.
If you add a name after the ':cl', that name will be used in the changelog. Leave it empty to use your GitHub name.

:cl:  
rscadd: Added cringe 
rscadd: *sheesh
rscadd: *lipsmack
rscadd: the following emotes now have sound
rscadd: *whistle, *iwhistle, *eyebrow
/:cl:

### CODE REVIEW

**Comment by cuackles** [comment]
please dont add sheeeesh it hurst me on a psycological level

**Comment by maxion12345** [comment]
> please dont add sheeeesh it hurst me on a psycological level

but what are greytiders if not edgy tiktok artists working at a minimum wage job

**Comment by ToasterBiome** [comment]
I have no words

**Comment by Anvilman6** [comment]
sheesh.ogg

cursed land

**Comment by maxion12345** [comment]
*dab *sheesh *eyebrow *spin *roll *dab *eyebrow

**Comment by patpol4** [comment]
time to get those macros ready

**Comment by Tipy1802** [comment]
While the adition of sheesh is questionable giving sounds to the other emotes it really good

**Comment by Mqiib** [comment]
No sheesh
please

**Comment by maxion12345** [comment]
> No sheesh please

new midround
time traveler
can *sheesh *dab *tpose *thud to cause brain damage to people nearby, eventually killing them
goals are to get to ian and scream what the dog doing

**Comment by Cartlord** [comment]
I despise *eyebrow unironically

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -16,8 +16,10 @@
 
 /datum/emote/living/carbon/human/eyebrow
 	key = "eyebrow"
+	key_third_person = "raises an eyebrow"
 	message = "raises an eyebrow."
-
+	message_param = "raises an eyebrow at %t."
+	sound = 'sound/voice/thud.ogg'
```
Comment (adamsong) [inline code/modules/mob/living/carbon/human/emote.dm]: I've never heard a noise when raising my eybrows

**Review by adamsong — CHANGES_REQUESTED** [review]

**Comment by SomeguyManperson** [comment]
I doagnose you with cringe

**Pull request closed without merging**